integration-tests/mongodb-client-ssl/src/test/java/io/quarkus/it/mongodb/SSLMongoTest.java

```diff
@@ -0,0 +1,64 @@
+package io.quarkus.it.mongodb;
+
+import static org.hamcrest.core.Is.is;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.jboss.logging.Logger;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+
+import de.flapdoodle.embed.mongo.MongodExecutable;
+import de.flapdoodle.embed.mongo.MongodStarter;
+import de.flapdoodle.embed.mongo.config.IMongodConfig;
+import de.flapdoodle.embed.mongo.config.MongodConfigBuilder;
+import de.flapdoodle.embed.mongo.config.Net;
+import de.flapdoodle.embed.mongo.distribution.Version;
+import de.flapdoodle.embed.process.runtime.Network;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+
+@DisabledOnOs(OS.LINUX)
```

Comment:
It would be great to get that tested on our CI (using linux).
@evanchooly any idea about this?